# fixes (bnc#438681)
# calling ped_exception_throw(*,*,NULL) causes endless loop allocating memory
# since it does not output any usefull data anyway, we can remove that call completely
--- ./libparted/labels/dasd.c.orig	2008-11-26 13:55:11.000000000 +0100
+++ ./libparted/labels/dasd.c	2008-11-26 13:55:49.000000000 +0100
@@ -211,7 +211,6 @@ ped_disk_dasd_done ()
 static int
 dasd_probe (const PedDevice *dev)
 {
-	char *errstr = 0;
 	LinuxSpecific* arch_specific;
 	struct fdasd_anchor anchor;
 
@@ -238,7 +237,6 @@ dasd_probe (const PedDevice *dev)
 
 error_cleanup:
 	fdasd_cleanup(&anchor);
-	ped_exception_throw(PED_EXCEPTION_ERROR,PED_EXCEPTION_IGNORE_CANCEL,errstr);
 
 	return 0;
 }
